tests/test_lightcurve.py

```diff
@@ -1021,8 +1022,10 @@ def test_flatten_returns_normalized():
     )
     flat_lc, trend_lc = lc.flatten(window_length=3, polyorder=1, return_trend=True)
 
-    assert flat_lc.flux.unit is lc_flux_unit
-    assert flat_lc.flux_err.unit is lc_flux_unit
+    assert flat_lc.flux.unit == u.dimensionless_unscaled
+    assert flat_lc.flux_err.unit == u.dimensionless_unscaled
+    assert flat_lc.meta["NORMALIZED"]
+    
```

Comment:
@keyuxing Minor style comments for future reference: please avoid having lines with spaces only (line 1028 here and line 825 in `lightcurve.py`, not visible in the github diff), or have trailing spaces at the end of lines, as trailing spaces could cause spurious diffs in subsequent commits.

Reply:
Thanks for your reminder! I'll pay attention to it afterwards.